hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1472,55 +1479,98 @@
       name: chip_entropy_src_ast_fips
       desc: '''Verify the connectivity of rng_fips_o feedback signal to RNG.
 
-            Details TBD.
+            - This signal is asserted at boot time when the entropy is generated at low latency.
+            - Add connectivity assertion check to verify connectivity of fips output to AST.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_entropy_src_csrng
-      desc: '''Verify the transfer of entropy bits to CSRNG.
+      name: chip_entropy_src_fuse_en_fw_read
+      desc: '''Verify the FW read fuse input entropy_src.
 
-            Verify the entropy valid interrupt.
-            At the CSRNG, validate the reception of entropy req interrupt.
-            Details TBD.
+            - Initialize the OTP with the entropy_src FW read fuse bit enabled.
+            - Program and enable the entropy_src in LFSR mode; program a known seed, enable the
+              observe fifo ready interrupt.
+            - Route the entropy data to register and disable the conditioning logic.
```

Comment:
Need to update this since LFSR mode is going to be completely removed, due to security risk.

Reply:
Removal is included in PR #8527